src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -4,6 +4,10 @@
 .header {
   border-bottom: 1px solid $orange;
   padding: 8px 0;
+  position: sticky;
+  top: -15px;
+  background-color: $black;
+  z-index: 1;
```

Comment:
probably should raise `.geneViewTabs` z-index to 2 then; so that in the unlikely case the header scrolls past the tab buttons it will go behind them

Reply:
That's done

Reply:
To `.geneViewTabs`, not to `.header` 😄 To avoid this:

![image](https://user-images.githubusercontent.com/6834224/81960803-687b3b00-9609-11ea-8fd9-acddeb841887.png)

![image](https://user-images.githubusercontent.com/6834224/81960832-716c0c80-9609-11ea-9aa0-4080203fe716.png)
 
![image](https://user-images.githubusercontent.com/6834224/81960856-7c26a180-9609-11ea-9654-c4989ab208f0.png)

Reply:
Ah! My bad. That's done.